### DISCUSSION TITLE
Icon request: toy-brick (lego-block)

### DISCUSSION THREAD

**Discussion by thelumberjhack** [FortAwesome/Font-Awesome#21138] [Icon Requests]
Hi, I'm building my personal web site. And I'm fond of Lego ^^
It would be great if you could had a lego block/blocks icon (![Lego block example](https://cloud.githubusercontent.com/assets/5294889/3142141/e9be87e0-e9ac-11e3-8826-67898eaf9d82.png) for example) in a future version, and of course their logo.

Thanx in advance

**Comment by stevenvh17** [comment]
I understand the hammer would represent "building", but IMO Lego doesn't need it; it's about building anyway. Besides, just a Lego building block could be bigger  and better recognizable on small scale.

**Comment by thelumberjhack** [comment]
I totally agree with you. Since it's recognizable as a lego block it will do the trick.
Thanks in advance (again) :+1:

**Comment by ensonic** [comment]
+1

**Comment by jbkunst** [comment]
+10!

**Comment by jkfoto** [comment]
+1

**Comment by craigiswayne** [comment]
+1

**Comment by psyhtest** [comment]
+1

**Comment by jverne** [comment]
1x1

**Comment by someidiot** [comment]
+1

**Comment by jan-molak** [comment]
+1

**Comment by cv6** [comment]
+1

**Comment by JonathanBeia** [comment]
+1

**Comment by pyeremenko** [comment]
:+1:

**Comment by joshgilson** [comment]
+100

**Comment by GoGurtPrime** [comment]
This would be a great way to represent add-ons / plugins on the site I'm building.

+(100^10)

**Comment by klonos** [comment]
A lego block would be a good match for the "Structure" menu item for the open source Backdrop CMS. We are currently using https://fontawesome.com/icons/sort-amount-down and perhaps we could use https://fontawesome.com/icons/sitemap which is what our Brethren in Drupal are using for the same menu item.

**Comment by latere-a-latere** [comment]
I really like the idea and see a lot of use in it but I suggest naming it "toy-brick" instead:

1. LEGO is a registered trademark
3. The correct term is brick, not block
4. There are various other brand selling similar bricks ever since LEGO lost the patent on theirs.

**Comment by GoGurtPrime** [comment]
Oh hey, looks like this went to the wrong recipient!

**Comment by GoGurtPrime** [comment]
NVM whoops, I'm old. Don't mind me. Carry on lol

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by SaltNLife143** [comment]
Zengaz.worldwide.toy brick? Lego  legends
